pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1884,6 +1884,7 @@ type TSDBSpec struct {
 	// in a breaking way.
 	//
 	// It requires Prometheus >= v2.39.0 or PrometheusAgent >= v2.54.0.
+	// +optional
 	OutOfOrderTimeWindow Duration `json:"outOfOrderTimeWindow,omitempty"`
```

Comment:
We need a pointer.
```suggestion
	OutOfOrderTimeWindow *Duration `json:"outOfOrderTimeWindow,omitempty"`
```

Reply:
Thanks @simonpasquier . 
Change it done.